auth-api/src/auth_api/services/flags.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Manage the Feature Flags initialization, setup and service."""
+import logging
```

Comment:
LD configuration setup the same as PAY now. so they're consistent. It was never used in auth.